source/src/revision.rs

```diff
@@ -113,7 +113,7 @@ pub fn remote<P, U>(
 where
     P: Clone + ToString,
 {
-    let remote_branches = branches(browser, Some(into_branch_type(Some(peer_id.clone()))))?;
+    let remote_branches = branches(browser, into_branch_selector(Some(peer_id.clone())))?;
```

Comment:
While we're at it, can we turn the `into_branch_selector` function into a `From<Option<PeerId>> for BranchSelector` instance?

Reply:
Ya good point, although it won't be – technically – PeerId, it'll be P